scripts/llama-canary-agent-repair.sh

```
@@ -419,6 +434,74 @@ run_battery() {
+  if [[ "$(git rev-parse HEAD)" != "${CERTIFIED_SHA:?}" || -n "$(git status --porcelain)" ]]; then
+    if [[ -n "$(git status --porcelain)" ]]; then
+      git add -A
+      if ! git diff --cached --quiet; then
+        git commit -m "review(llama): agent review fixes at upstream ${UPSTREAM_SHA:0:10}" \
+          -m "Modifications from the post-certification agent review of the canary repair (see the repair PR comment for the review report)."
+      fi
+    fi
+    echo "post-green review made changes; publishing review head $(git rev-parse --short HEAD)"
+    git push "$(repair_remote)" "+HEAD:refs/heads/${BRANCH}" 2> >(redact_token >&2) \
+      || echo "warning: could not push the review commit; the certified tree remains the PR head" >&2
+    REVIEW_STATUS="$(printf ' Post-green agent review made modifications: PR head is now the review commit `%s` (certified commit `%s` above; review changes re-certify when this PR merges and the next canary runs).' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)